internal/verifier/migration_verifier.go

```diff
@@ -136,6 +136,8 @@ type Verifier struct {
 	globalFilter map[string]any
 
 	pprofInterval time.Duration
+
+	changeEventRecheckBuf ChangeEventRecheckBuffer
```

Comment:
Suggestion: We shouldn’t need this if we switch to reading all events from the cursor into a slice.